coba/learners/regCB.py

```diff
@@ -0,0 +1,219 @@
+import math
+
+from coba.utilities import PackageChecker
+from coba.simulations import Context, Action
+from coba.learners.core import Learner, Key
+from typing import Any, Dict, Sequence
+
+from copy import deepcopy
+
+class RegCB(Learner):
+    """A learner using the RegCB algorithm from "Practical Contextual Bandits with Regression Oracles" by Foster et al.
```

Comment:
Do you mind adding a little more explanation explaining the relationship between the two citations? That is the algorithm is due to Foster while the online bin search implementation is due to Bietti?